src/MaRRTPathPlanNode.py

```diff
@@ -45,14 +42,24 @@ def __init__(self):
         if rospy.has_param('~filename'):
             self.filename = rospy.get_param('~filename')
 
+        if rospy.has_param('~odom_topic'):
+            self.odometry_topic = rospy.get_param('~odom_topic')
+        else:
+            self.odometry_topic = "/odometry"
+
+        if rospy.has_param('~world_frame'):
+            self.world_frame = rospy.get_param('~world_frame')
+        else:
+            self.world_frame = "world"
+
         waypointsFrequency = rospy.get_param('~desiredWaypointsFrequency', 5)
         self.waypointsPublishInterval = 1.0 / waypointsFrequency
         self.lastPublishWaypointsTime = 0
 
         # All Subs and pubs
-        rospy.Subscriber("/map", Map, self.mapCallback)
-        rospy.Subscriber("/odometry", Odometry, self.odometryCallback)
-        # rospy.Subscriber("/car_sensors", CarSensors, self.carSensorsCallback)
+        rospy.Subscriber("/track", Track, self.mapCallback)
```

Comment:
Once it is all about track (instead of map), lets have a new callback name too (self.trackCallback)